pkgs/servers/klipper/klipper-genconf.nix

```diff
@@ -0,0 +1,21 @@
+{ writeShellApplication
+, klipper
+, python2
+, gnumake
+, pkgsCross
+}: writeShellApplication {
+  name = "klipper-genconf";
+  runtimeInputs = [
+    python2
+    pkgsCross.avr.stdenv.cc
+    gnumake
+  ];
+  text = ''
+    CURRENT_DIR=$(pwd)
+    TMP=$(mktemp -d)
+    make -C ${klipper.src} OUT="$TMP" KCONFIG_CONFIG="$CURRENT_DIR/config" menuconfig
```

Comment:
This doesn't look right to me, menuconfig is going to run an interactive application during a build.

Reply:
Yes, see my answer above. This is just a standalone helper script.